src/modules/tickets/TicketController.ts

```diff
@@ -0,0 +1,50 @@
+import "reflect-metadata";
+import {
+  Body,
+  Delete,
+  Get,
+  JsonController,
+  Param,
+  Post,
+  Put,
+  Patch,
+} from "routing-controllers";
+import { ITicket } from "../../entities/ITicket";
+import { TicketService } from "./TicketService";
+import Container from "typedi";
+import { UpdateTicketStatus, updateTicket } from "./dto/TicketRequest";
+
+@JsonController()
+export class TicketController {
+  ticketService = Container.get(TicketService);
+
+  @Get("/tickets")
+  getAll() {
+    return this.ticketService.getAllTicket();
+  }
+
+  @Get("/tickets/:id")
+  getOne(@Param("id") id: string) {
+    return this.ticketService.getTicket(id);
+  }
+
+  @Post("/tickets")
+  post(@Body() ticket: ITicket) {
+    return this.ticketService.addTicket(ticket);
+  }
+
+  @Put("/tickets/:id")
+  put(@Param("id") id: string, @Body() body: updateTicket) {
```

Comment:
updateTicket should be class as class-validator and class should have suffix Request and Pascal case
```suggestion
  put(@Param("id") id: string, @Body() body: UpdateTicketRequest) {
```